### ISSUE TITLE
loss_two always zero

### ISSUE THREAD

**Issue by PeteBai** [BingfengYan/CO-MOT#24]
I tried to train the model with BDD100K dataset. During training loss_two remains zero while other indicators vary through epochs. I am unable to find exactly what it is through the code and the paper, as there ares no metrics that contain "_two_". Can you shed some light regarding the functionality of this loss?

**Comment by fengxiuyaun** [comment]
Can you show more information, such as a screenshot?

**Comment by PeteBai** [comment]
in engine.py, line 71-80

![Image](https://github.com/user-attachments/assets/0d1716a6-0c37-41e9-b6f9-c94b1c5ae5c8)

**Comment by fengxiuyaun** [comment]
https://github.com/BingfengYan/CO-MOT/blob/a1c96640ecf46b4fabd5850c86eee2d0d0b847a7/engine.py#L71
This is normal, because two is not used.
